README.md

```diff
@@ -25,7 +25,7 @@ Currently requires node 8+
 
 `fixtures.mock(scenario)` will intercept requests using [nock](https://www.npmjs.com/package/nock).
 `scenario` is a String in the form `<host name>/<scenario name>`. `host name`
-is any folder in [`fixtures/`](fixtures/). `scenario name` is any filename in
+is any folder in [`scenarios/api.github.com/`](scenarios/api.github.com/). `scenario name` is any filename in
```

Comment:
Thanks! The correct folder is `scenarios/`, `api.github.com/` is already the host name that is referred to in the text

Reply:
@gr2m Done ;)